Fix VM Size link text

bheesma opened the pull request:
Link text seems to point to Linux VM Sizes, but it could point to either Windows or Linux VM sizes based on the URL. Hence changed to reflect title of the page obtained from left side menu.

azurecla commented:
Hi __@laoluolapegba__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

rmca14 commented:
Sent for author approval.

bheesma committed:
Fix VM Size link text

Link text seems to point to Linux VM Sizes, but it could point to either Windows or Linux VM sizes based on the URL. Hence changed to reflect title of the page obtained from left side menu.
